### PR TITLE
Update dependencies + add Dependabot

### PR DESCRIPTION
**Pull request by mgrabovsky**

### CODE REVIEW

**Comment by mgrabovsky** [comment]
Note: I haven't tested this thoroughly yet. I've tried a couple of GraphQL queries and all worked fine. I didn't try waiving – that should be checked before merging this.

File: package.json [hunk 1]
```diff
@@ -27,51 +27,50 @@
...
-    "kerberos": "^1.1.7",
+    "kerberos": "^2.0.1",
     "lodash": "^4.17.21",
     "moment": "^2.29.4",
     "mongodb": "^4.11.0",
     "node-cron": "^3.0.2",
     "pako": "^2.1.0",
-    "passport": "^0.5.3",
-    "passport-saml": "^3.2.4",
+    "passport": "^0.6.0",
```
Comment (Andrei-Stepanov) [inline package.json:41]: With this we could not login previous time.

https://github.com/jaredhanson/passport/issues/939
It was necessary to revert.
Please do not update this package to 0.6.0 until ^^^ is fixed.

File: package.json [hunk 2]
```diff
@@ -27,51 +27,50 @@
...
+    "@node-saml/passport-saml": "^4.0.2",
```
Comment (Andrei-Stepanov) [inline package.json:42]: Same here, I would update these 2 packages togather.

**Comment by mgrabovsky** [comment]
@Andrei-Stepanov I've removed the `passport`-related upgrades and updated the PR.

**Comment by Andrei-Stepanov** [comment]
I like it, if possible please send in new PR a part for improving auth part in small PR.
It is independent from packages update, It will be easier to deal with if something.

**Comment by mgrabovsky** [comment]
@Andrei-Stepanov Updated.

**Review by Andrei-Stepanov — APPROVED** [review]

**Pull request merged**